.NET/Microsoft.Recognizers.Text.DateTime/Spanish/Extractors/SpanishDateTimePeriodExtractorConfiguration.cs

```diff
@@ -2,17 +2,61 @@
 using System.Text.RegularExpressions;
 
 using Microsoft.Recognizers.Definitions.Spanish;
-using Microsoft.Recognizers.Text.Number;
 
 namespace Microsoft.Recognizers.Text.DateTime.Spanish
 {
-    public class SpanishDateTimePeriodExtractorConfiguration : BaseOptionsConfiguration, IDateTimePeriodExtractorConfiguration
+    public class SpanishDateTimePeriodExtractorConfiguration : BaseOptionsConfiguration,
+        IDateTimePeriodExtractorConfiguration
     {
-        public static readonly Regex NumberCombinedWithUnit = new Regex(DateTimeDefinitions.DateTimePeriodNumberCombinedWithUnit, RegexOptions.IgnoreCase | RegexOptions.Singleline);
+        public static readonly Regex NumberCombinedWithUnit =
+            new Regex(DateTimeDefinitions.DateTimePeriodNumberCombinedWithUnit,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
-        private static readonly Regex FromRegex = new Regex(DateTimeDefinitions.FromRegex, RegexOptions.IgnoreCase | RegexOptions.Singleline);
-        private static readonly Regex ConnectorAndRegex = new Regex(DateTimeDefinitions.ConnectorAndRegex, RegexOptions.IgnoreCase | RegexOptions.Singleline);
-        private static readonly Regex BetweenRegex = new Regex(DateTimeDefinitions.BetweenRegex, RegexOptions.IgnoreCase | RegexOptions.Singleline);
+        private static readonly Regex FromRegex = new Regex(DateTimeDefinitions.FromRegex,
+            RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        private static readonly Regex ConnectorAndRegex = new Regex(DateTimeDefinitions.ConnectorAndRegex,
+            RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        private static readonly Regex BetweenRegex = new Regex(DateTimeDefinitions.BetweenRegex,
+            RegexOptions.IgnoreCase | RegexOptions.Singleline);
+
+        //TODO: add this according to the related part in English
+        public static readonly Regex WeekDayRegex =
+            new Regex(@"^[\.]", RegexOptions.IgnoreCase | RegexOptions.Singleline);

```

Comment:
These should read from the definition file. Can you move the placeholder regex to the YAML file?